crates/config/src/config.rs

```diff
@@ -392,6 +392,34 @@ impl PruneConfig {
     pub fn has_receipts_pruning(&self) -> bool {
         self.segments.receipts.is_some() || !self.segments.receipts_log_filter.is_empty()
     }
+
+    /// Merges another `PruneConfig` into this one, taking values from the other config iff the
+    /// corresponding value in this config is not set.
```

Comment:
```suggestion
    /// Merges another `PruneConfig` into this one, taking values from the other config if the
    /// corresponding value in this config is not set.
```

Reply:
better write verbose "if and only if" @garwahl